inject.js

```diff
@@ -191,7 +191,7 @@ chrome.runtime.sendMessage({}, function(response) {
     window.onload = () => {
       initializeNow(window.document)
     };
-    if (document && document.doctype && document.doctype.name == "html") {
```

Comment:
Why is this being removed? This was added to address https://github.com/igrigorik/videospeed/issues/309

Reply:
I think that test makes the initialization not happen when the URL is smth like file:///C:/Users/myvideo.mp4 (?)

Reply:
https://i.imgur.com/DeJZjiN.png

Reply:
Ah, interesting.. doh. Yeah, let's find a different solution for #309 then. Could you take a look at the page flagged in #309 and see if we can find a different solution for it?

Reply:
I think my solution is not breaking it but might break it. why is it here?
https://github.com/igrigorik/videospeed/commit/7606224ec92790f65f27b348c9fb7346ecbc7d3e